src/ztex_md5crypt.c

```diff
@@ -66,11 +66,11 @@ static struct device_bitstream bitstream = {
 	{ 2, 14336, 4094 },
 	// computing performance estimation (in candidates per interval)
 	// (keys * mask_num_cand)/crypt_all_interval per jtr_device.
-	147456,		// keys/fpga for crypt_all()
-	16384,		// keys/fpga for self-test
-	460800,		// Would be ~15 MB of USB traffic on 32-byte keys
+	32*12*640,	// keys/fpga for crypt_all()
+	8192,		// keys/fpga for self-test
+	600*1024,	// Would be ~15 MB of USB traffic on 32-byte keys
```

Comment:
Does the comment about "~15 MB of USB traffic" possibly need to be updated along with this change? To "~20 MB"? Ditto for phpass.